db/migrate/20171104051300_create_join_table_users_meet_options.rb

```diff
@@ -0,0 +1,8 @@
+class CreateJoinTableUsersMeetOptions < ActiveRecord::Migration[5.0]
+  def change
+    create_join_table :users, :meet_options do |t|
```

Comment:
can you explain what this migration is for?

Reply:
When a new user registers, there is the "Availability to Meet for" field in the form with options such as "Coffee", "Mentorship", ... listed as check-boxes. This is a many-to-many relationship, as users can have many meet options and meet options can have many users.  Therefore, there has to be a join table to make the association between the User tables and the Meet_Options table. Hence the join table Users_Meet_Options.